examples/backlist_both_tokens.py

```diff
@@ -0,0 +1,48 @@
+from flask_jwt_extended import decode_token
+
+
+# Endpoint for loggin in
+@app.route('/login', methods=['POST'])
+def login():
+    username = request.json.get('username', None)
+    password = request.json.get('password', None)
+    if username != 'test' or password != 'test':
+        return jsonify({"msg": "Bad username or password"}), 401
+
+    user = {id: 1, username: 'test'}
+    refresh_token = create_refresh_token(identity=user)
+
+    # Embed the refresh token's jti in the access_token
+    user["refresh_jti"] = decode_token(refresh_token)["jti"]
```

Comment:
There is a helper function for this: https://flask-jwt-extended.readthedocs.io/en/latest/api.html#flask_jwt_extended.get_jti